src/main/java/uk/gov/justice/digital/domains/model/TableTuple.java

```diff
@@ -1,17 +1,18 @@
 package uk.gov.justice.digital.domains.model;
 
+import lombok.Data;
+import java.util.regex.PatternSyntaxException;
+
+@Data
 public class TableTuple {
     private String schema;
     private String table;
 
-    private String originalSchema;
-    private String originalTable;
-
-    public TableTuple() {
+    private final String originalSchema;
+    private final String originalTable;
 
-    }
 
-    public TableTuple(final String source) {
+    public TableTuple(final String source) throws PatternSyntaxException {
         if(source != null && source.contains(".")) {
             this.schema = source.split("\\.")[0];
```

Comment:
No need to split the string twice. Do `val elements = source.split("\\.");` and then set the values from this.